### PR TITLE
Wire sign up and login UI to backend APIs.

### PR DESCRIPTION
**Pull request by thomasvandoren**
signup-flow

Update sign up flow so it passes through the pass/teentix.org menu
(SignUpOptionsController).

Calls AccountManager.CreateAccount() when user taps the
agree-to-terms-button. Assuming create account is successful, go to new
VerifyEmailController that instructs user to go check their email and open the
verify link it includes.

Note: the copy in the verify email controller probably needs work. I just put
in some text as a placeholder for now.

When the user taps Login! button on verify email screen, it attempts the login
and if successful segues to the How It Works screen.

login-flow

For login, it calls AccountManager.Login() when user taps on button. When login
is successful, segue to new screen that just shows the session. This is just a
placeholder for now that shows the login was successful, because I wasn't sure
where login should land.

There are empty helper methods defined on the login, terms, and verify email
controllers that can be used to disable/enable the forms and show/hide
spinners.

### CODE REVIEW

File: TeenTixCommon/RestAPI.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace TeenTix.Common
+{
+	public class RestAPI
+	{
+		public static string Session { get; set; }
+
+		public RestAPI ()
+		{
+		}
+	}
```
Comment (thomasvandoren) [inline TeenTixCommon/RestAPI.cs]: This class will contain the all the api calls that are not related to account creation/login, like get calendar/favorites/etc.

**Comment by lava10** [comment]
LOOKS AMAZING!
